### ISSUE TITLE
Embedding Layer fails to Import from Keras 1.2.2 to DL4J 0.9.1

### ISSUE THREAD

**Issue by mobiusinversion** [deeplearning4j/deeplearning4j#5063]
Hello, 

Trying to load a Keras 1.2.2 model into DL4J 0.9.1, but I get this:    

     org.deeplearning4j.nn.modelimport.keras.UnsupportedKerasConfigurationException: DL4J EmbeddingLayer includes bias but Keras Embedding does not. Please file an issue at http://github.com/deeplearning4j/deeplearning4j/issues.

	at org.deeplearning4j.nn.modelimport.keras.layers.KerasEmbedding.<init>(KerasEmbedding.java:67)
	at org.deeplearning4j.nn.modelimport.keras.KerasLayer.getKerasLayerFromConfig(KerasLayer.java:238)
	at org.deeplearning4j.nn.modelimport.keras.KerasModel.helperPrepareLayers(KerasModel.java:191)

The Keras model is defined as follows:

    model = Sequential()
    model.add(Embedding(len(word_index) + 1, embedding_size,
                    weights=[embedding_matrix],
                    input_length=MAX_SEQUENCE_LENGTH,
                    trainable=True))
    model.add(Dropout(0.2))
    model.add(LSTM(256))
    model.add(Dense(Y.shape[1], activation='softmax'))
    model.compile(loss='binary_crossentropy', optimizer='adam', metrics=['accuracy'])

**Comment by maxpumperla** [comment]
yes, I think this has been addressed in 0.9.2 and further improved in current snaphots.

**Comment by mobiusinversion** [comment]
@maxpumperla , where can I get artifacts for 0.9.2?
